plugins/trace_api_plugin/include/eosio/trace_api/trace.hpp

```diff
@@ -21,6 +21,17 @@ namespace eosio { namespace trace_api {
       chain::bytes                        data = {};
    };
 
+   // the purpose of action_version reserve some space for future change,  
+   struct action_trace_v1 {
```

Comment:
This should inherit from `action_trace_v0`.  The data type is purely additive for the field at the end.

Reply:
will try.